File: app/src/main/java/com/whiteside/insta/ui/NoInternet.java
package com.whiteside.insta.ui;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

import com.whiteside.insta.R;

import Controller.Connection;

public class NoInternet extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_no_internet);
    }

    public void checkInternet(View view) {
        if(Connection.isConnected(this)){
            finish();
        }
    }
}

File: app/src/main/java/com/whiteside/insta/NoInternet.kt
package com.whiteside.insta

import com.whiteside.insta.model.Connection
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.whiteside.insta.R.*

class NoInternet : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(layout.activity_no_internet)
    }

    fun checkInternet(view: View?) {
        if (Connection.isConnected(this)) {
            finish()
        }
    }
}